tests/update_skll_models.py

```diff
@@ -33,6 +33,7 @@ def update_model(model_file):
 
     # now load the SKLL model
     model = Learner.from_file(model_file)
+    model.logger = ''
```

Comment:
Can we add a comment here that we won’t need this once we update SKLL to v1.5.1? I will also file an issue so that we remember.